### PR TITLE
Minimal tests

### PR DESCRIPTION
**Pull request by huguesdevimeux**
closes #28

### CODE REVIEW

**Comment by leotrs** [comment]
Oh also please add (closes #28) to your PR description so github knows to close that issue automatically when this is merged.

**Comment by huguesdevimeux** [comment]
@leotrs Yes, you're absolutely right about the LaTex thing. I am changing that.

And which file are you talking about? I don't see any `sample_scenes.py`. 
If you were talking of `example_scenes.py` I would be in favor of keeping it.

**Comment by PgBiel** [comment]
We're still going to have to test LaTeX usage though

Also if you're using TextMobject then it's implied it will use LaTeX; you'll want to replace everywhere where it says "TextMobject" with "Text" in order to avoid LaTeX

**Comment by kilacoda** [comment]
We could implement a check in `manimlib.imports` for that I think. Something like
```py
if latex_installed():
    from manimlib.mobject.types.svg.tex_mobject import *
```

**Comment by leotrs** [comment]
@huguesdevimeux yes I did mean example_scenes.py, my bad.

My rationale is that it is highly unorthodox to have `.py` files at the top level. I'd much rather have a new folder `samples/` that includes that file, or just use those scenes in `example_scenes.py` as tests (as you are kinda doing right now). I really don't think it's a good idea to leave `example_scenes.py` up top.

**Comment by leotrs** [comment]
Ack, sorry to be fussy, I meant, to move the file to `samples/samples_scenes.py` and not to `manimlib
samples/samples_scenes.py`. Two reasons for this: the sample scenes are in principle not part of the core library, and in fact can be ran without knowledge of any of the other code in `manimlib/`. The other reason is that the `sample_scenes.py` file needs to `import manimlib` and that gets tricky when inside the manimlib folder itself. (In fact, we are trying to keep all imports local within the library itself as per #15.)

**Comment by huguesdevimeux** [comment]
Done, I createed a folder `samples/`

Nevertheless I don't think its a good idea to mix up `samples_scenes.py `with testing. My idea is that in `tests_example_scene.py`, one would be able to do precise test about a precise feature without having to render a full and complex scene like there are/there will be in `samples_scenes.py`. At the moment the tests functions are not plentiful because we didn't start the feature development, but I'm convinced that it will be soon or later. 

And thank you for being "fussy" like you said, it helps me to do less mistakes :)

**Comment by leotrs** [comment]
Fully agreed there, the point is to have *something* that *could* serve as a test, even if it's just a run-these-scenes-and-eyeball-them kind of a test. In the (hopefully near) future, that will most definitely change to something way more exhaustive.

So for now, we can use the scenes in `samples/` as quick and dirty tests, until we have a true `tests/` folder.

**Comment by leotrs** [comment]
Great! We need one more review here. @PgBiel ?

File: tests/tests_example_scene.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+from manimlib.imports import *
+
```
Comment (yoshiask) [inline tests/tests_example_scene.py:2]: Shouldn't this file be named `testing_sample_scenes.py` or something to be consistent with the other sample file?

Comment (huguesdevimeux) [reply tests/tests_example_scene.py]: Yup. Changed

**Pull request merged**